docs/technical-reference.md

```diff
@@ -256,44 +256,55 @@ users to be able to sign up.
 
 ## Single sign-on (SSO)
 
+> You will need SSO administrator privileges to add the Quilt application.
+
```

Comment:
Eliminate. It's implied and "admin" means different things on different platforms.